src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -3132,4 +3133,52 @@ export default class EthereumApi implements Api {
     return "2";
   }
   //#endregion
+
+  //#region txpool
+
+  /**
+   * Returns the current content of the transaction pool.
+   *
+   * @returns The transactions currently pending or queued in the txpool.
+   * @example
+   * ````javascript
+   * const [from] = await provider.request({ method: "eth_accounts", params: [] });
+   * const pendingTx = await provider.request({ method: "eth_sendTransaction", params: [{ from, gas: "0x5b8d80", nonce:"0x0" }] });
+   * const queuedTx = await provider.request({ method: "eth_sendTransaction", params: [{ from, gas: "0x5b8d80", nonce:"0x2" }] });
+   * const pool = await provider.send("txpool_content");
+   * console.log(pool);
+   * ````
+   */
+  @assertArgLength(0)
+  async txpool_content() {
+    const { transactions, common } = this.#blockchain;
+    const { transactionPool } = transactions;
+
+    const processMap = (map: Map<string, Heap<TypedTransaction>>) => {
+      let res = {};
+      for (let [_, transactions] of map) {
+        if (transactions === undefined) continue;
```

Comment:
```suggestion
```

I know this was taken from the `transaction-pool.ts`'s find method, but this line isn't actually needed, as it should not be possible for `transactions` to ever be `undefined` here.